### PR TITLE
LVL Glitchy-Hero

### PR DESCRIPTION
**Pull request by mckelvey**
This PR adds the basic internals for the glitchy Level hero.

### CODE REVIEW

**Comment by mckelvey** [comment]
[Storybook > Level Protocol](https://trello.com/c/jXrPXk6v/23-storybook-level-protocol)

File: __stories__/Hero.stories.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+}
+
+const Story = {
+  title: 'Sections / Hero',
+  component: Hero,
+  argTypes: {
+    glitchSpeed: {
+      description: 'The glitch speed is the milliseconds between iterations.',
+      table: {
+        type: {
```
Comment (gustavoalfaro) [inline __stories__/Hero.stories.js:62]: Great documentation!

File: components/hooks/useSimpleObserver.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react'
+
+const useSimpleObserver = (ref, callback, deps) => {
+  useEffect(() => {
+    let observer = null
+    let el = null
+
+    if (ref.current) {
+      el = ref.current
+      observer = new IntersectionObserver(
```
Comment (gustavoalfaro) [inline components/hooks/useSimpleObserver.js:10]: Pretty new for me!

File: components/hooks/useSimpleObserver.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      observer.observe(el)
+    }
+
+    return () => {
+      if (observer && el) {
+        observer.unobserve(el)
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [callback, ref, ...deps])
```
Comment (gustavoalfaro) [inline components/hooks/useSimpleObserver.js:28]: What if you just pass deps not spreading?, not sure about what could happen, but I'm curious

File: components/ui/Hero/Hero.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+  GLITCH_SETTLED_THRESHOLD,
+  GLITCH_SPEED,
+  INITIAL_GLITCHY_NESS,
+} from './glitchy-ness'
+
+const HeroSection = styled.section`
+  display: grid;
+  justify-items: center;
+  grid-template-columns: 1fr;
+  grid-gap: 0.8rem;
```
Comment (gustavoalfaro) [inline components/ui/Hero/Hero.js:25]: Yup `rem` for the win!

File: components/ui/Hero/glitchy-ness.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+// The de-glitch factor is how fast the glitchy-ness cleans itself up, per iteration.
```
Comment (gustavoalfaro) [inline components/ui/Hero/glitchy-ness.js:1]: Love the documentation here!

**Review by gustavoalfaro — APPROVED** [review]
amazing

**Review by gustavoalfaro — APPROVED** [review]
amazing

**Pull request merged**